tests/settings.py

```diff
@@ -102,6 +102,9 @@
 if 'RESOLWE_DOCKER_COMMAND' in os.environ:
     FLOW_DOCKER_COMMAND = os.environ['RESOLWE_DOCKER_COMMAND']
 
+# Ignore errors in tests when pulling Docker images from 'list_docker_images --pull' command
+FLOW_DOCKER_IGNORE_PULL_ERRORS = strtobool(os.environ.get('RESOLWE_DOCKER_IGNORE_PULL_ERRORS', '1'))
```

Comment:
Why do you default to `'1'` here and also explicitly set this to `1` in `.travis.yml`?

Reply:
The setting in the Travis config is redundant, I'll remove it.